src/test/java/com/synopsys/integration/alert/web/api/job/JobConfigActionsTest.java

```diff
@@ -516,30 +525,11 @@ public void validateJobsByIdEmptyListTest() {
         assertTrue(actionResponse.hasContent());
     }
 
-    @Test
-    public void validateJobsByIdInternalServerErrorTest() throws Exception {
-        JobIdsRequestModel jobIdsRequestModel = new JobIdsRequestModel(List.of(jobId));
-        DescriptorKey descriptorKey = createDescriptorKey();
-        Descriptor descriptor = createDescriptor(descriptorType);
-
-        Mockito.when(descriptorMap.getDescriptorMap()).thenReturn(Map.of(descriptorKey, descriptor));
-        Mockito.when(authorizationManager.anyReadPermission(Mockito.any())).thenReturn(true);
-        Mockito.when(jobAccessor.getJobsById(Mockito.any())).thenReturn(List.of(configurationJobModel));
-
-        Mockito.doThrow(new AlertException("Exception for Alert test")).when(fieldModelProcessor).performAfterReadAction(Mockito.any());
-
-        ActionResponse<List<JobFieldStatuses>> actionResponse = jobConfigActions.validateJobsById(jobIdsRequestModel);
-
-        assertTrue(actionResponse.isError());
-        assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, actionResponse.getHttpStatus());
-        assertFalse(actionResponse.hasContent());
-    }
-
```

Comment:
Why is this no longer needed?

Reply:
The method `JobConfigActions.convertJobDatabaseModelToRestModel(...)`, the method that threw the exception this test was for, is no longer used in `JobConfigActions.validateJobsById(...)`. So there is no longer a need to test exception handling as there is none.